src/app/modules/dashboard/pages/dashboard/components/cpu-stats/cpu-stats.component.ts

```diff
@@ -0,0 +1,116 @@
+import {Component, OnInit, OnDestroy, ViewChild} from '@angular/core';
+import {Color, BaseChartDirective} from 'ng2-charts';
+import {timer, BehaviorSubject, Subject, Subscription} from 'rxjs';
+import {takeUntil} from 'rxjs/operators';
+import {ToastrService} from 'ngx-toastr';
+
+import {DashboardMetricsService} from '../../../../services/dashboard-metrics.service';
+
+@Component({
+    selector: 'app-cpu-stats-component',
+    templateUrl: './cpu-stats.component.html',
+    styleUrls: ['./cpu-stats.component.scss']
+})
+export class CpuStatsComponent implements OnInit, OnDestroy {
+    @ViewChild(BaseChartDirective, {static: false})
```

Comment:
Because of my comment below
```suggestion
    @ViewChild(BaseChartDirective, {static: true})
```